migration/v0.26.x-v0.27.x/reinstall-hnc.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+set -e
+
+here="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+
+"${here}/../../bin/ck8s" ops helm wc uninstall hnc -n hnc-system
+"${here}/../../bin/ck8s" ops kubectl wc delete hncconfigurations.hnc.x-k8s.io config
+"${here}/../../bin/ck8s" ops helmfile wc -l group=hnc apply
```

Comment:
Just to be sure, this does not remove any of the subnamespaces that users might have created, right?

Reply:
Correct, removing hnc does not remove the subnamespaces